src/core/utils/wrap_stdbool.h

```diff
@@ -44,7 +44,7 @@
 /* c++ has a built in bool */
 #else
 
-#if defined(_MSC_VER)
+#if HAVE__BOOL
```

Comment:
This is not a must-have for me. As I mentioned in the title, this is cosmetic for me: i.e. I don't require it for my compilations.